packages/edit-site/src/components/dataviews/README.md

```diff
@@ -0,0 +1,107 @@
+# DataView
+
+This file aims to document the main APIs related to the DataView component.
+
+## View
+
+The view is responsible for configuring how the dataset is visible to the user. For example:
+
+```js
```

Comment:
Do you think we should also add the type definitions for the interfaces used in the main component or somewhere, now?

Reply:
At some point, it'd be good to have them, sure.

Right now, my main concern is to have a high-level _map of the code_, so to speak. I'd like to give people a little orientation about how to navigate the hundreds of lines of code. Writing things down is how I think, so I am biased towards believing this may be helpful to some others as well.